### PR TITLE
New version: py-yarl 1.7.2

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Review by scheibelp — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-yarl/package.py
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-yarl/package.py:22]: This should be `when='@1.7.2 ^python@:3.7'`

**Review by adamjstewart — COMMENTED** [review]
@scheibelp thanks for the review, but remember to allow package maintainers to review PRs before merging

**Pull request merged**
